src/test/ui/issues/issue-87490.rs

```diff
@@ -0,0 +1,9 @@
+trait StreamOnce {  //~ ERROR `main` function not found in crate
```

Comment:
Can you add a `fn main() {}` to this test to avoid this irrelevant error? That way this test doesn't need to get updated when the main function error message changes.